### DISCUSSION TITLE
How to know if PG_TIMETABLE is running ?

### DISCUSSION THREAD

**Discussion by msisolucoes** [cybertec-postgresql/pg_timetable#170] [Q&A] [answered]
How to know if PG_TIMETABLE is running ?

**Comment by pashagolub** [comment]
How about 
```console
ps -A | grep pg_timetable
```

**Reply by msisolucoes** [reply]
Hi,

Thank you very much for your answer, but I think I expressed myself, I will rephrase.
I wonder if there is a form by command SQL, example:
select timetable.isrunning;
Thank you.

**Comment by pashagolub** [comment, marked as answer]
Ah, you mean from the database point of view :) Sorry.

Well the most obvious would be to check the special table. You can read about this in [my blog post](https://www.cybertec-postgresql.com/en/pg_timetable-v3-is-out/).

```sql
SELECT * FROM timetable.active_session;
```

You can, of course, check special PostgreSQL view, in case you are using outdated `pg_timetable` (I hope you're not :-)
```sql
SELECT * FROM pg_stat_activity WHERE datname = 'mydatabasename';
```

Kind regards!
